src/simularium/rendering/InstancedFiberShader.ts

```diff
@@ -264,7 +264,30 @@ void createTube (float t, vec2 volume, out vec3 offset, out vec3 normal) {
 
   // compute the TBN matrix
   vec3 T = normalize(next - prev);
-  vec3 B = normalize(cross(T, next + prev));
+  vec3 B = vec3(0,0,0);
+  // if normalize(next-prev) and next+prev are parallel, then B will be zero
+  float check = dot(next-prev, next+prev);
```

Comment:
nit, inconsistent spacing around operators , not sure why the linter didn't pick this up

Reply:
this doesn't get linted because it is a long inline string written in webGL